Sources/tinys3/model/Multipart/PartSizeCalculator.swift

```diff
@@ -7,23 +7,28 @@ enum PartSizeCalculator {
     static let typicalNumberOfParts = 500
 
     static func calculate(basedOn fileSize: Int) -> Int {
-        max(min(fileSize, minPartSize), min(fileSize / typicalNumberOfParts, maxPartSize))
+        max(minPartSize, min(fileSize / typicalNumberOfParts, maxPartSize))
```

Comment:
ℹ️  This change now means that for files smaller than the `minPartSize` (5MB). `calculate` will always return 5MB (while before it returned the fileSize for those cases)… even if in practice for those small files, they will be constituted of a single part, whose size will be the entire size of the original file.

That doesn't change how multipart uploads will split the data in the end, because for small files (<5MB) the result of `PartSizeCalculator` was never really used in the final computation (e.g. for a file size of 3kb, `slice(from: 0, to: 3kb, by: 5kb).map { … }` will return a single part whose range will be `0..<3kb` and the part size of 5kb will not be used due to the `min(start + partSize, fileSize)` used in the `slice(…).map { … }`)